module-gui/gui/widgets/status-bar/SignalStrengthBar.cpp

```diff
@@ -25,7 +25,7 @@ namespace gui::status_bar
         constexpr auto signal3_roaming = "gsm_3_roaming";
         constexpr auto signal4_roaming = "gsm_4_roaming";
 
-        using SignalMap = std::unordered_map<Store::RssiBar, std::string>;
+        using SignalMap = std::map<Store::RssiBar, std::string>;
```

Comment:
What is the reason for changing `unordered_map` over `map`?

Reply:
There was some very strange error during linking stage regarding `ceill` method (multiple symbol definition), by analyzing map file I found out that the only module that uses this function is somewhere in `SignalStrengthBar.cpp` and it turned out to be used in `unordered_map` container. It's not a proper fix, I'm aware of it, but I've run out of other ideas to resolve this issue. This map is rather small, so this change should not affect the performance noticeably.

Reply:
Ok, that's strange. Thanks for the investigation and explanation.